src/index.ts

```diff
@@ -13,49 +13,80 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
 import * as extend from 'extend';
 import * as path from 'path';
-import {AuthenticationConfig, Common, ServiceConfig} from '../third_party/types/common-types';
-import {Config, defaultConfig} from './config';
+const gcpMetadata = require('gcp-metadata');
```

Comment:
Can you use import?

Reply:
I don't think so -- gcpMetadata doesn't have type definitions.

Reply:
But it does, as of very recently: https://github.com/stephenplusplus/gcp-metadata/pull/9

Reply:
Okay -- I think I can now that I've updated the version used.